packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -128,17 +128,16 @@ export const messages = {
     'Deleting source files deletes the files from your computer and removes the corresponding metadata from your default org. Are you sure you want to delete this source from your project and your org?',
   confirm_delete_source_button_text: 'Delete Source',
   cancel_delete_source_button_text: 'Cancel',
-  force_analytics_template_create_text:
-    'SFDX: Create Sample Analytics Template',
-  force_analytics_template_name_text: 'template name',
-  force_apex_class_create_text: 'SFDX: Create Apex Class',
-  force_apex_unit_class_create_text: 'SFDX: Create Apex Unit Test Class',
-  force_visualforce_component_create_text: 'SFDX: Create Visualforce Component',
-  force_visualforce_page_create_text: 'SFDX: Create Visualforce Page',
-  force_lightning_app_create_text: 'SFDX: Create Aura App',
-  force_lightning_component_create_text: 'SFDX: Create Aura Component',
-  force_lightning_event_create_text: 'SFDX: Create Aura Event',
-  force_lightning_interface_create_text: 'SFDX: Create Aura Interface',
+  analytics_generate_template_text: 'SFDX: Create Sample Analytics Template',
+  analytics_template_name_text: 'template name',
```

Comment:
Unrelated to SFDX -> SF: These two are only available in English, not in Japanese.  i18n.ja.ts does not contain translations for these messages, nor are they directly copied over as English messages.